.github/workflows/openstack-ansibleee-runner.yaml

```diff
@@ -0,0 +1,68 @@
+name: OpenStackAnsibleEE runner image builder
+
+on:
+  push:
+    branches:
+      - '*'
+    paths-ignore:
+      - .gitignore
+      - .pull_request_pipeline
+      - LICENSE
+      - README.rst
+      - .github/
+      - docs/
+      - tests/
+
+env:
+  imageregistry: 'quay.io'
+  imagenamespace: ${{ secrets.IMAGENAMESPACE || secrets.QUAY_USERNAME }}
+  latesttag: latest
+
+jobs:
+
+  check-secrets:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Check secrets are set
+        id: have-secrets
+        if: "${{ env.imagenamespace != '' }}"
+        run: echo "::set-output name=ok::true"
```

Comment:
Github has deprecated `::set-output`, read [here](https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/)

You can use `run: echo "ok=true" >> $GITHUB_OUTPUT`